packages/io/dxf-deserializer/instantiate.js

```diff
@@ -364,107 +324,70 @@ function instantiateMesh (obj, layers, options) {
   } else {
     // invalid vlen
   }
-  return CSG.fromPolygons(polygons)
-}
-
-//
-// find the layer referenced by the given object
-//
-function findLayer (obj, layers) {
```

Comment:
not needed either ?

Reply:
Moved to helpers.js, which eliminates duplicate code.

Reply:
ok good !